pip/wheel.py

```diff
@@ -37,6 +37,12 @@
 from pip.utils.temp_dir import TempDirectory
 from pip.utils.ui import open_spinner
 
+try:
+    from concurrent.futures.process import ProcessPoolExecutor
```

Comment:
I mean technically the core library could be changed, but isn't that subject to a backward compatibility guarantee?